src/assets/wise5/services/studentPeerGroupService.ts

```diff
@@ -146,4 +146,30 @@ export class StudentPeerGroupService extends PeerGroupService {
       return of([peerGroupStudentData]);
     }
   }
+
+  retrieveStudentWork(
+    peerGroup: PeerGroup,
+    nodeId: string,
+    componentId: string,
+    showWorkNodeId: string,
+    showWorkComponentId: string
+  ): Observable<any> {
+    if (this.configService.isPreview()) {
+      const studentWork = [];
+      const latestComponentState = this.studentDataService.getLatestComponentStateByNodeIdAndComponentId(
+        showWorkNodeId,
+        showWorkComponentId
+      );
+      if (latestComponentState != null) {
+        studentWork.push(latestComponentState);
+      }
+      return of(studentWork);
```

Comment:
Use ternary expression to simplify code?
```suggestion
      const latestComponentState = this.studentDataService.getLatestComponentStateByNodeIdAndComponentId(
        showWorkNodeId,
        showWorkComponentId
      );
      return latestComponentState != null ? of([latestComponentState]) : of([]);
```